src/glue/resources/table_columns.yaml

```diff
@@ -494,6 +494,8 @@ tables:
         Type: string
       - Name: SleepLogDetails
         Type: array<struct<Type:string,StartDate:string,EndDate:string,Value:string>>
+      - Name: IsMainSleep
```

Comment:
I don't see this having been mentioned in the jira ticket? Is there something else driving this change specifically?

Reply:
I'm combining two tickets into one. Here's the other ticket: https://sagebionetworks.jira.com/browse/ETL-586

Both tickets involve minor changes to the same file.